spec/immutable-data-structures.html

```diff
@@ -49,8 +49,10 @@ <h1>Record.fromEntries ( _iterable_ )</h1>
           1. Perform ? RequireObjectCoercible(_iterable_).
           1. Let _fields_ be a new empty List.
           1. Let _adder_ be a new Abstract Closure with parameters (_key_, _value_) that captures _fields_ and performs the following steps when called:
+            1. If Type(_key_) is Symbol, throw a *TypeError* exception.
             1. If Type(_value_) is Object, throw a *TypeError* exception.
-            1. Let _field_ be { [[Key]]: _key_, [[Value]]: _value_ }.
+            1. Let _keyString_ be ? ToString(_key_)
```

Comment:
Doesn’t this already handle the Symbol case?

Reply:
yes, but we figured out that for symmetry with Object.fromEntries, we should use ToPropertyKey instead and then the assertion for a symbol becomes needed again